src/machine-learning/machine-learning-api-client.ts

```diff
@@ -54,13 +54,23 @@ export interface AutoMLTfliteModelOptions extends ModelOptionsBase {
     automlModel: string;
   };
 }
-export type ModelOptions = ModelOptionsBase | GcsTfliteModelOptions | AutoMLTfliteModelOptions;
+export interface GcsCoremlModelOptions extends ModelOptionsBase {
+  coremlModel: {
+    gcsCoremlUri: string;
+  };
+}
+export type ModelOptions = ModelOptionsBase | GcsTfliteModelOptions | AutoMLTfliteModelOptions | GcsCoremlModelOptions;
 export type ModelUpdateOptions = ModelOptions & { state?: { published?: boolean }};
 
 export function isGcsTfliteModelOptions(options: ModelOptions): options is GcsTfliteModelOptions {
   const gcsUri = (options as GcsTfliteModelOptions)?.tfliteModel?.gcsTfliteUri;
   return typeof gcsUri !== 'undefined'
 }
+export function isGcsCoremlModelOptions(options: ModelOptions): options is GcsCoremlModelOptions {
```

Comment:
Please add spacing around declarations. Definitely around all functions, and ideally interfaces too.

Reply:
done